demisto_sdk/commands/common/content/tests/objects/pack_objects/playbook/playbook_test.py

```diff
@@ -27,16 +22,3 @@ def test_objects_factory():
 def test_prefix():
     obj = Playbook(PLAYBOOK)
     assert obj.normalize_file_name() == PLAYBOOK.name
-
-
-def test_playbook_summary():
```

Comment:
why delete?

Reply:
I reverted my previous version + added some new handling on the new so I deleted the tests from my previous version.